infra/common/src/test/java/cn/hippo4j/common/toolkit/Md5UtilTest.java

```diff
@@ -50,12 +54,15 @@ public void assetEncodeHexString() {
 
     @Test
     public void assetGetTpContentMd5() {
-        String md5Result = "ef5ea7cb47377fb9fb85a7125e76715d";
-        ThreadPoolParameterInfo threadPoolParameterInfo = ThreadPoolParameterInfo.builder().tenantId("prescription")
-                .itemId("dynamic-threadpool-example").tpId("message-consume").content("描述信息").corePoolSize(1)
-                .maximumPoolSize(2).queueType(1).capacity(4).keepAliveTime(513L).executeTimeOut(null).rejectedType(4)
-                .isAlarm(1).capacityAlarm(80).livenessAlarm(80).allowCoreThreadTimeOut(1).build();
-        Assert.isTrue(md5Result.equals(Md5Util.getTpContentMd5(threadPoolParameterInfo)));
+        final ThreadPoolParameterInfo threadPoolParameterInfo = new ThreadPoolParameterInfo();
+        final String mockContent = "mockContent";
+        final String mockContentMd5 = "34cf17bc632ece6e4c81a4ce8aa97d5e";
```

Comment:
The previous md5 value was for "threadPoolParameterInfo" object. Previously there was no mocking, so that was the real md5 value for the JSON string generated for that object. Now I mocked " ContentUtil.getPoolContent" method to return string "mockContent". The current md5 value is for "mockContentMd5".

Reply:
Got you